tests/unit-tests/test_decoder.py

```diff
@@ -1010,6 +1055,8 @@ def test_EntityRegression(in_dim, out_dim):
     test_EntityRegression(8, 1)
     test_EntityRegression(8, 8)
 
+    test_EntityClassifier(64, 8)
```

Comment:
- This PR is for decoder bias, but there is no test case for it.
- The `test_EntityClassifier` has multiple input combination, should we test more combination here?